admin_guide/manage_nodes.adoc

```diff
@@ -77,30 +77,77 @@ $ oc describe node <node>
 For example:
 
 ----
-$ oc describe node node1.example.com
-Name:			node1.example.com
-Labels:			kubernetes.io/hostname=node1.example.com
-CreationTimestamp:	Wed, 10 Jun 2015 17:22:34 +0000
-Conditions:
-  Type		Status	LastHeartbeatTime			LastTransitionTime			Reason					Message
-  Ready 	True 	Wed, 10 Jun 2015 19:56:16 +0000 	Wed, 10 Jun 2015 17:22:34 +0000 	kubelet is posting ready status
-Addresses:	127.0.0.1
-Capacity:
- memory:	1017552Ki
- pods:		100
- cpu:		2
-Version:
- Kernel Version:		3.17.4-301.fc21.x86_64
- OS Image:			Fedora 21 (Twenty One)
- Container Runtime Version:	docker://1.6.0
- Kubelet Version:		v0.17.1-804-g496be63
- Kube-Proxy Version:		v0.17.1-804-g496be63
-ExternalID:			node1.example.com
-Pods:				(2 in total)
-  docker-registry-1-9yyw5
-  router-1-maytv
-No events.
-----
+# oc describe node vm252-51.gsslab.pnq2.redhat.com 
```

Comment:
@mburke5678 - hey - can you please take out names of internal Red Hat instances and replace with generic hostnames throughout this file?

Reply:
@vikram-redhat PTAL to make sure I got all instances
3.4 https://github.com/openshift/openshift-docs/pull/10820
3.5 https://github.com/openshift/openshift-docs/pull/10821
3.6 https://github.com/openshift/openshift-docs/pull/10822
3.7 https://github.com/openshift/openshift-docs/pull/10823 
3.9 https://github.com/openshift/openshift-docs/pull/10824

Reply:
@mburke5678 - looks good, but why not do it in master and then CP? Unless this changes never made it to the master?

Reply:
@vikram-redhat Each version has different output for the command; some very different. I did each version from a separate branch in my repo and merged to the given version

Reply:
@mburke5678 - ah makes sense. What about master and 3.10 though?

Reply:
@vikram-redhat Good thoughts on master. Should the 3.9 output go to master? 

I will ask again about 3.10. I asked previously, but did not get a response.

Reply:
@mburke5678 master is the latest and greatest. You could run this on Origin using oc cluster up for Origin and test it that way to see what the output is.

Reply:
@vikram-redhat I merged to master via https://github.com/openshift/openshift-docs/pull/10837